src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -489,6 +489,31 @@ typedef struct {
     DOMString decoderImplementation; //!<  TODO Identifies the decoder implementation used. This is useful for diagnosing interoperability issues.
 } RtcInboundRtpStreamStats, *PRtcInboundRtpStreamStats;
 
+/**
+ * Reference: https://www.w3.org/TR/webrtc/#dom-rtcdatachannelstate
+ */
+enum RTCDataChannelState {
+    RTC_DATA_CHANNEL_STATE_CONNECTING,
+    RTC_DATA_CHANNEL_STATE_OPEN,
+    RTC_DATA_CHANNEL_STATE_CLOSING,
+    RTC_DATA_CHANNEL_STATE_CLOSED
+};
+
+/**
+ * Reference: https://www.w3.org/TR/webrtc-stats/#dom-rtcdatachannelstats
+ */
+struct {
+    DOMString label;                //!< The "label" value of the RTCDataChannel object.
+    DOMString protocol;             //!< The "protocol" value of the RTCDataChannel object.
+    long dataChannelIdentifier;     //!< The "id" attribute of the RTCDataChannel object.
+    DOMString transportId;          //!< A stats object reference for the transport used to carry this datachannel.
+    enum RTCDataChannelState state; //!< The "readyState" value of the RTCDataChannel object.
```

Comment:
Can we typedef the enum and also have the enum type in all caps? 
`RTC_DATA_CHANNEL_STATE`

Reply:
yes